index.js

```diff
@@ -0,0 +1,179 @@
+const fs = require('fs');
+
+const rainbowLib = require('rainbow-bridge-lib');
+const { BorshContract } = require('rainbow-bridge-lib/rainbow/borsh');
+const {
+    EthProofExtractor,
+    receiptFromWeb3,
+    logFromWeb3,
+} = require('rainbow-bridge-lib/eth-proof-extractor');
+const { rlp } = require('ethereumjs-util')
+
+const utils = rainbowLib.utils;
+const nearAPI = rainbowLib.nearAPI;
+
+const NEAR_CONNECTOR_DEPOSIT = nearAPI.utils.format.parseNearAmount('50');
+const NEAR_CONNECTOR_NEW_GAS = '100000000000000';
+const ETH_CONNECTOR_DEPLOY_GAS = 5000000;
+const ETH_BRIDGE_TOKEN_DEPLOY_GAS = 5000000;
+const NEAR_BRIDGE_TOKEN_DEPOSIT = nearAPI.utils.format.parseNearAmount('40');
+const NEAR_BRIDGE_TOKEN_DEPLOY_GAS = '100000000000000';
+
+const NEAR_CONNECTOR_SCHEMA = {
+    bool: {
+        kind: 'function',
+        // @ts-ignore
+        ser: (b) => Buffer.from(Web3.utils.hexToBytes(b ? '0x01' : '0x00')),
+        deser: (z) => readerToHex(1)(z) === '0x01',
+    },
+    Proof: {
+        kind: 'struct',
+        fields: [
+            ['log_index', 'u64'],
+            ['log_entry_data', ['u8']],
+            ['receipt_index', 'u64'],
+            ['receipt_data', ['u8']],
+            ['header_data', ['u8']],
+            ['proof', [['u8']]],
+        ],
+    },
+}
+
+async function createNearConnector(masterAccount, config) {
```

Comment:
It seems like there are two possible use-cases for this function, and I want to see if I understand them correctly:

1. from a server/command line environment, where you have a proper "master account" that can create a new sub-account and deploy a connector contract
2. a frontend app that will pass in the signed-in user's account and a `nearConnectorId` for an existing connector

Please correct this understanding if it's off

I'm trying to figure out what values to pass to this from my frontend app at https://github.com/near-examples/erc20-to-nep21/pull/1/files